lib/rspec/rails/example/channel_example_group.rb

```diff
@@ -0,0 +1,93 @@
+require "rspec/rails/matchers/action_cable/have_streams"
+
+module RSpec
+  module Rails
+    # @api public
+    # Container module for channel spec functionality. It is only available if
+    # ActionCable has been loaded before it.
+    module ChannelExampleGroup
+      # @private
+      module ClassMethods
+        # These blank modules are only necessary for YARD processing. It doesn't
+        # handle the conditional check below very well and reports undocumented objects.
+      end
+    end
+  end
+end
+
+if ::Rails::VERSION::MAJOR >= 6
```

Comment:
Should use the feature checks

Reply:
See https://github.com/rspec/rspec-rails/pull/2131